### DISCUSSION TITLE
Blank import

### DISCUSSION THREAD

**Discussion by aherranzh** [microsoft/powercat-creator-kit#246] [Q&A]
Hi all,

I have installed the Creator Kit solution, but when I try to import the component library it does not recognize any in the environment. 

![image](https://user-images.githubusercontent.com/110382552/209951973-e8812f60-07d4-4ac9-b256-641956ddb9d1.png)

As additional information I also have the Center of Excellence solution installed, which in its latest version requires having the Creator Kit installed.

Has this happened to anyone else?

**EDIT**: I'm the owner of all the solutions, so they are shared with me (including the component library)

Thanks in advance
Regards

**Comment by denise-msft** [comment]
@aherranzh are you still having issues with the latest release?

**Comment by denise-msft** [comment]
Closing this as inactive. If you still have trouble with this, please re-post this in issues section.
